geoextent/lib/handleCSV.py

```diff
@@ -10,18 +11,31 @@
                    "latitude" : ["(.)*latitude(.)*", "^lat","lat$", "^y","y$"],
            "time":["(.)*timestamp(.)*", "(.)*datetime(.)*", "(.)*time(.)*", "date$","^date"]}
 
-def checkFileValidity(filePath):
+def checkFileValidity(filepath):
     '''Checks whether it is valid CSV or not. \n
     input "path": type string, path to file which shall be extracted \n
     raise exception if not valid
     '''
-    logger.info("Checking validity of {} \n".format(filePath))
-    
-    with open(filePath) as csv_file:
-        daten = csv.reader(csv_file.readlines())
-        if daten is None:
-            logger.error("File {} is invalid!".format(filePath))
-            raise Exception("The file {} has no valid csv Attributes".format(filePath))
+
+    logger.info(filepath)
+    try:
+        file = gdal.OpenEx(filepath)
+        driver = file.GetDriver().ShortName
+    except:
+        logger.debug("File {} is NOT supported by HandleCSV module".format(filepath))
+        return False
+
+    if driver == "CSV":
```

Comment:
Hmmm... so you're telling me, this will work just as well as using `csv.reader(..)` to detect CSV files? Are there no problems with different delimiters etc? Just asking :-)